masterlist.yaml

```
@@ -6572,3 +6572,24 @@ plugins:
     clean:
       - crc: 0x17ED1071
         util: '[FO4Edit v4.0.4](https://www.nexusmods.com/fallout4/mods/2737)'
+
+  - name: 'VtawWardrobe5.esp'
+    url:
+      - link: 'https://www.nexusmods.com/fallout4/mods/43844/'
+        name: 'Vtaw Wardrobe 5 - CBBE - BodySlide (Nexus Mods)'
+    msg:
+      - <<: *patch3rdParty
+        subs:
+          - '[Fusion Girl](https://www.loverslab.com/files/file/18238-zex-fusion-girl/)'
```

Sorry, I forgot to mention this in previous PRs, but you don't need to link to the mod if it can be assumed that the user already has it:

```suggestion
          - 'Fusion Girl'
```